src/components/metadata/MetaSimple.js

```diff
@@ -7,24 +7,12 @@ import moment from 'moment';
 import momentLocalizer from 'react-widgets/lib/localizers/moment';
 import StaticIndex from '../../containers/views/StaticIndex';
 import Icon from '../Icon';
+import FilePicker from '../FilePicker';
 import 'react-widgets/dist/css/react-widgets.css';
 
 momentLocalizer(moment);
 
 export class MetaSimple extends Component {
-  state = {
-    staticfiles: [],
```

Comment:
Why were using this and not using it now?

Reply:
It was a development artifact. If you inspect the [commit](https://github.com/jekyll/jekyll-admin/commit/9531f8966e3a2cd4a6299a07cc3ed4dbf0d666d0) that introduced this, and [the file directly in the final unified diff](https://github.com/jekyll/jekyll-admin/blob/ead9cffc087bf135e9db17e3ea2be81564042bf8/src/components/metadata/MetaSimple.js), you'll see that `{this.state.staticfiles}` and `{this.fetchStaticFiles}` had already become redundant.